x-pack/auditbeat/processors/sessionmd/docs/add_session_metadata.asciidoc

```diff
@@ -1,6 +1,89 @@
 [[add-session-metadata]]
-=== Add session metadata
+=== Add session metadata (Beta)
 
-PLACEHOLDER
+beta::[]
 
+++++
+<titleabbrev>add_session_metadata</titleabbrev>
+++++
 
+ifeval::["{beatname_lc}"=="auditbeat"]
```

Comment:
The conditions are set in the processor list, so you do not want to set them here.
```suggestion
```